src/components/RadioButton/RadioButton.js

```diff
@@ -114,8 +114,13 @@ const RadioButtonLabel = styled('label')`
  */
 const RadioButton = ({ onToggle, children, name, ...props }) => (
   <Fragment>
-    <RadioButtonInput id={name} type="radio" {...{ ...props, name }} />
-    <RadioButtonLabel htmlFor={name} onClick={onToggle} {...{ ...props }}>
+    <RadioButtonInput
+      id={name}
+      type="radio"
+      onClick={onToggle}
+      {...{ ...props, name }}
```

Comment:
Yup, here you have to do this, because you're adding props to the existing `props` object before spreading.